src/components/TodoList/TodoList.scss

```diff
@@ -0,0 +1,46 @@
+.item-enter {
```

Comment:
Consider using nested selectors to better organize the code. For example, group the `.item-*` and `.temp-item-*` classes under their respective parent classes.